Bottom: 1bcbefb425fbee88fc3e451039a3a9822decb97f
Top:    acc85403600456599aa0bed3f04d8c7dd155621f
Author: Tiwei Bie <tiwei.bie@intel.com>
Date:   2018-04-25 13:15:50 +0800

virtio_ring: enable packed ring

Signed-off-by: Tiwei Bie <tiwei.bie@intel.com>


---

diff --git a/drivers/virtio/virtio_ring.c b/drivers/virtio/virtio_ring.c
index e4b9be5..77369e3 100644
--- a/drivers/virtio/virtio_ring.c
+++ b/drivers/virtio/virtio_ring.c
@@ -1955,6 +1955,8 @@ void vring_transport_features(struct virtio_device *vdev)
 			break;
 		case VIRTIO_F_IOMMU_PLATFORM:
 			break;
+		case VIRTIO_F_RING_PACKED:
+			break;
 		default:
 			/* We don't understand this bit. */
 			__virtio_clear_bit(vdev, i);
